starter-code/src/Logos.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import './Logos.css'
+
+class Logos extends Component {
```

Comment:
Again, remember that a component represents a single reusable thing, so its better to use a singular name.